### ISSUE TITLE
need to design a tree store for large studies/trees that are integrated into OpenTree via scripts 

### ISSUE THREAD

**Issue by mtholder** [OpenTreeOfLife/germinator#111]
@josephwb has a supertree study with >1000 trees.
@snacktavish will soon have automatically generated gene trees.
Brian O'Meara has some very large trees for which he has/wants to have the OTU mapping done with scripts.

These don't really fit nicely into the design of phylesystem as data store for manually curated trees (most of which are presumably intended for synthesis).

Huge studies will make us hit our git repo limit sooner (we can use shards then, but still a bit of an annoyance). Plus they won't load in the curator app (they'll require to much RAM).

Automatically curated trees are a generated product that probably should not be versioned.

So basically we need a data store that:
  1. is updatable,
  2. plays well with synthesis,
  3. makes it easy for users to find the trees, 
  4. but which does not put everything into one NexSON that is pulled down by the curator app.

**Comment by mtholder** [comment]
Just to offer one vague suggestion: We could have tiny NexSON stub for the study that lives in phylesystem and has the reference info, other associated data, and (in some cases) perhaps the OTU mapping.  And then just have links to newicks that are stored elsewhere. 

We'd need to work on a syntax for augmenting the raw newick with extra info (e.g. ingroup and perhaps interpretation of branch lengths).  But it shouldn't be too hard to do that.

**Comment by josephwb** [comment]
Probably not relevant, but most of my trees are tiny (there are just a lot of them):
![rplot](https://cloud.githubusercontent.com/assets/808499/18783692/489d35a2-815b-11e6-8492-45a6151b7f9d.png)

**Comment by snacktavish** [comment]
+1 to the nexson stub idea, especially for the published large trees. and maybe a fully separate repo for the automatically generated trees that haven't been peer reviewed. I'll think on this!

**Comment by jar398** [comment]
s3 storage was $.03 / Gb / month last I checked.  The command line tool for
reading and writing is called 's3cmd'. I think the files are readable via
HTTP, but not sure how.

**Comment by jar398** [comment]
Perhaps of interest: https://help.github.com/articles/about-git-large-file-storage/

**Comment by jar398** [comment]
A use case from @bomeara: https://github.com/OpenTreeOfLife/opentree/issues/788
